mne/viz/topomap.py

```diff
@@ -2116,6 +2116,22 @@ def plot_evoked_topomap(
     :ref:`gridspec <matplotlib:arranging_axes>` interface to adjust the colorbar
     size yourself.
 
+    The defaults for ``contours`` and ``vlim`` are handled as follows:
+
+    * When neither ``vlim`` nor a list of ``contours`` is passed, MNE sets
+    ``vlim`` at ± the maximum absolute value of the data and then chooses
+    contours within those bounds.
+
+    * When ``vlim`` but not a list of ``contours`` is passed, MNE chooses
+    contours to be within the ``vlim``.
+
+    * When a list of ``contours`` but not ``vlim`` is passed, MNE chooses
+    ``vlim`` to encompass the ``contours`` and the maximum absolute value of the
+    data.
+
+    * When both a list of ``contours`` and ``vlim`` are passed, MNE uses them
+    as-is.
```

Comment:
```suggestion
    * When neither ``vlim`` nor a list of ``contours`` is passed, MNE sets
      ``vlim`` at ± the maximum absolute value of the data and then chooses
      contours within those bounds.

    * When ``vlim`` but not a list of ``contours`` is passed, MNE chooses
      contours to be within the ``vlim``.

    * When a list of ``contours`` but not ``vlim`` is passed, MNE chooses
      ``vlim`` to encompass the ``contours`` and the maximum absolute value of the
      data.

    * When both a list of ``contours`` and ``vlim`` are passed, MNE uses them
      as-is.
```

Reply:
I think @drammock is right and this should fix it.

The Azure Windows failures are likey unrelated though. I'll look into fixing them separately in a different PR